sdk/identity/azure-identity/src/default_azure_credential.cpp

```diff
@@ -26,7 +26,18 @@ using Azure::Core::_internal::StringExtensions;
 using Azure::Core::Diagnostics::Logger;
 using Azure::Identity::_detail::IdentityLog;
 
+namespace {
+std::string const EnvVarName = "AZURE_TOKEN_CREDENTIALS";
```

Comment:
`EnvVarName` is far too generic - the name should reflect the purpose of the environment variable, not just that it's the name of a random environment variable.